config/spryk/spryks/Spryker/Zed/Business/AddCrudFacade.yml

```diff
@@ -210,3 +210,11 @@ postSpryks:
                       value: "is_active"
                   propertyType:
                       value: "BOOLEAN"
+    
+    -   AddZedBusinessFacadeTest:
```

Comment:
Fix indentation and too many whitespaces should be `- A` one whitespace.

Reply:
done